frontend/src/components/UserDashboard/ChangePassword/api.js

```diff
@@ -0,0 +1,18 @@
+export const changePasswordAPI = async (values, actions) => {
+  console.log(values);
+
+  const url = `http://localhost:8080/changePassword/${values.id}?password=${values.password}`;
+  try {
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error(`Response status: ${response.status}`);
+    }
+    const json = await response.json();
+    actions.setStatus({status:"success", data:json});
+    console.log(json);
+  } catch (error) {
+    console.error(error.message);
+  }
+  actions.setStatus("fail");
```

Comment:
Thank you for catching that; you saved me debugging time. I had edited it without testing since it’s related to the API part, but I’ll make sure to fix it when connecting the frontend and backend.